docs/getting-started/integrate-auth/react-with-elements.mdx

```diff
@@ -0,0 +1,384 @@
+
+If you don't already have the latest Node.js LTS, install it from here [Node.js LTS](https://nodejs.org/en/download/)
+
+To interact with Ory's APIs we install the [Ory SDK](https://www.npmjs.com/package/@ory/client)
+
+To use the Ory Element features, we install [Ory Elements](https://www.npmjs.com/package/@ory/elements)
+
+To manage our routes more dynamically, we install [React Router](https://www.npmjs.com/package/react-router-dom)
+
+```bash
+npm i @ory/client @ory/elements react-router-dom --save
+```
+
+To install Ory CLI follow [this guide](https://www.ory.sh/docs/guides/ory-cli-install-use)
+
+### Why do I Need the Ory CLI?
+
+```mdx-code-block
+import OryCLI from '../_common/ory-cli.mdx'
+
+<OryCLI />
+```
+
+```mdx-code-block
+import SdkEnvVar from '@site/src/components/SdkEnvVar'
+
+<SdkEnvVar />
+```
+
+### Create .env
+
+If you'd prefer to store your credentials in a .env file, you can do so by creating a .env file in the root of your project and
+adding the following:
+
+```bash
+VITE_ORY_SDK_URL=http://localhost:4000
+```
+
+### package.json
+
+At this point, your package.json should look something like this:
+
+```json
+...
+{
+  "dependencies": {
+    "@ory/client": "^0.2.0-alpha.48",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-router-dom": "^6.4.2"
+  }
+}
+...
+```
+
+### Create sdk.ts
+
+This file is used to create a new instance of the Ory SDK. We will use this instance to interact with the Ory APIs.
+
+```typescript
+import { V0alpha2Api, Configuration } from "@ory/client"
+
+export default new V0alpha2Api(
+  new Configuration({
+    //https://vitejs.dev/guide/env-and-mode.html#env-files
+    basePath: import.meta.env.VITE_ORY_SDK_URL,
+    baseOptions: {
+      withCredentials: true,
+    },
+  }),
+)
+```
+
+### Create main.tsx
+
+This file is used as the main canvas for React. We will use this file to create our routes and render our components.
+
+```tsx
+import React from "react"
+import ReactDOM from "react-dom/client"
+import { BrowserRouter } from "react-router-dom"
+import { ThemeProvider } from "@ory/elements"
+import App from "./App"
+import "./index.css"
+
+// import our Ory elements css
+import "@ory/elements/style.css"
+
+ReactDOM.createRoot(document.getElementById("root") as HTMLElement).render(
+  <React.StrictMode>
+    <BrowserRouter>
+      {/* We add the Ory themes here */}
+      <ThemeProvider>
+        <App />
+      </ThemeProvider>
+    </BrowserRouter>
+  </React.StrictMode>,
+)
+```
+
+If you are implementing Ory Elements in an existing project, you can just add the Ory Elements theme and imports instead as shown:
+
+```tsx
+//...imports
+import ReactDOM from "react-dom/client"
+import { BrowserRouter } from "react-router-dom"
+import { ThemeProvider } from "@ory/elements"
+//...
+
+//...existing main.tsx
+// Router from React Router
+;<BrowserRouter>
+  {/* We add the Ory themes here */}
+  <ThemeProvider>
+    <App />
+  </ThemeProvider>
+</BrowserRouter>
+//...
+```
```

Comment:
We can use a codeblock instead I think 
https://github.com/ory/docs/blob/master/docs/getting-started/integrate-auth/vue.mdx?plain=1#L50